smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -116,6 +123,15 @@ function makeReaderNameForDisplaying(readerName) {
   return readerName;
 }
 
+function updateAddDeviceButtonText(readersCount){
```

Comment:
nit: Add a space before `{`.

See the example formatting in the guideline: https://google.github.io/styleguide/jsguide.html#formatting